css/templates/travel/page-vars.css

```diff
@@ -112,3 +137,95 @@
   @custom-media --breakpoint-md-lg (min-width: 52.06rem) and (max-width: 72rem);
   @custom-media --breakpoint-v-md-lg (min-height: 52.06rem) and (max-height: 72rem);
 }
+
+body {
```

Comment:
most of this rule is already declared in -https://github.com/ampproject/ampstart/blob/master/css/ampstart-base/base-page.css#L23

May be all you need to do is just set the right variables.  (please dont declare new variables in places where you dont need it )